packages/calcite-components/src/components.d.ts

```diff
@@ -9,93 +9,193 @@ import { Alignment, Appearance, CollapseDirection, FlipContext, IconType, Kind,
 import { RequestedItem } from "./components/accordion/interfaces";
 import { IconNameOrString } from "./components/icon/interfaces";
 import { RequestedItem as RequestedItem1 } from "./components/accordion-item/interfaces";
+import { ActionMessages } from "./components/action/assets/action/t9n";
```

Comment:
I'm never sure - should we be committing these `components.d.ts` when there aren't related changes files or not?

Reply:
Yeah I'm not sure either. Sometimes it commits it for me when I don't want it to.